### PR TITLE
Integrate virtualized_is_guest with infrastructure_type. Closes #949

### LINKED ISSUE #949: Summary report virtualized data cleanup
## Specify type:
- Enhancement

## Description:

- combine virtualized_is_guest with infrastructure
- if present in VMware we know its a VM etc.

## Acceptance Criteria: 

- [ ] Verify that we only have an infrastructure value
- [ ] Verify that all vmware systems are marked as virtualized

____
Associated with: https://github.com/quipucords/quipucords/issues/947
___

### Expected behavior:
Describe the expected behavior for the program.

### Actual behavior:
Provide the actual behavior for the program.

### Steps to reproduce:
List clear steps to reproduce the bug.

### Environment information:
Provide detailed information on the *scanning* and *target* systems (if applicable); see example below.

Type            | Operating System | OS Version | Python Version | Virtualization
--------------- | ---------------- | ---------- | -------------- | --------------
Scanning System | RHEL             | 7.4        | 3.6            | Bare Metal
Target System   | RHEL             | 5.9        | 2.4.6          | RHEV

### PR DESCRIPTION
**Pull request by abaiken**

### CODE REVIEW

File: quipucords/fingerprinter/__init__.py [hunk 1]
```diff
@@ -755,10 +750,6 @@ def _process_vcenter_fact(source, fact):
     add_fact_to_fingerprint(source, 'vcenter_source', fact,
                             'infrastructure_type', fingerprint,
                             fact_value='virtualized')
-    add_fact_to_fingerprint(source, 'vcenter_source', fact,
```
Comment (kholdaway) [inline quipucords/fingerprinter/__init__.py]: Convert the values here to an infrastructure_type of `bare metal` or `virtualized`

File: quipucords/fingerprinter/__init__.py [hunk 2]
```diff
@@ -847,13 +838,6 @@ def _process_satellite_fact(source, fact):
                             'infrastructure_type', fingerprint,
                             fact_value=infrastructure_type)
 
-    virtualized_is_guest = bool(
```
Comment (kholdaway) [inline quipucords/fingerprinter/__init__.py]: Convert the values here to an infrastructure_type of `bare metal` or `virtualized`

File: quipucords/fingerprinter/__init__.py (hunk 1 above)
Comment (abaiken) [reply quipucords/fingerprinter/__init__.py]: The fact that I removed was always set to `True` and the line directly above sets the infrastructure type to `virtualized`.  If the virtualized_is_guest was always `True`, how do I decide between `bare metal` and `virtualized`?

File: quipucords/fingerprinter/__init__.py [hunk 3]
```diff
@@ -847,13 +838,19 @@ def _process_satellite_fact(source, fact):
...
-                            fact_value=virtualized_is_guest)
-
+    virtualized_is_guest = \
+        fact.get('virt_type') and \
+        fact.get('virtual_host') != fact.get('hostname')
+    if virtualized_is_guest:
+        infrastructure_type = 'virtualized'
+    elif virtualized_is_guest is False:
+        infrastructure_type = 'bare metal'
+    else:
```
Comment (kholdaway) [inline quipucords/fingerprinter/__init__.py]: Remove this and leave it unset.

**Pull request merged**